single_tech_samples/synapseanalytics/sample3_integration_testing/tests/integrationtests/data/dim_date.csv

```diff
@@ -0,0 +1,100 @@
+dim_date_id,date,day,day_suffix,week_day,week_day_name,DOW_in_month,day_of_year,week_of_month,week_of_year,ISO_week_of_year,month,month_name,quarter,quarter_name,year,MMYYYY,month_year,first_day_of_month,last_day_of_month,first_day_of_quarter,last_day_of_quarter,first_day_of_year,last_day_of_year,first_day_of_next_month,first_day_of_next_year
```

Comment:
If we have this file here for the test I don't think we need them as part of the deploy folder

Reply:
Removed.